contrib/local/configs/relayer-keys.json

```diff
@@ -24,7 +24,7 @@
     {
       "name": "persistenceCore",
       "id": "core-1",
-      "hdpath": "m/44'/750'/0'/0/0",
+      "hdpath": "m/44'/118'/0'/0/0",
       "keys": [
         {
           "name": "rly-core0",
```

Comment:
relayer addresses and pubkeys should also change in this file

Reply:
updated the addresses. pubkeys should not change

Reply:
thanks, now it seems ok

Reply:
now it seems ok, thanks!